src/main/resources/bamm/meta-model/2.0.0/aspect-meta-model-shapes.ttl

```diff
@@ -966,6 +1001,34 @@ bamm:AmbiguousEntityShape
       """
    ] .
 
+bamm:DoubleInstantiationShape
+   a sh:NodeShape ;
+   sh:target [
+      a sh:SPARQLTarget ;
+      sh:prefixes bamm:prefixDeclarations ;
+      sh:select """
+         select $this
+         where {
+           $this rdf:type/rdf:type ?bammElement .
+         }
+      """
+   ] ;
+   sh:sparql [
+      a sh:SPARQLConstraint ;
+      sh:message "{$this} can not be an instance of {?value}, because {?value} is an instance itself." ;
```

Comment:
Please add single quotes around the templates. This will be consistent in style with the rest of the messages.
Depending on what values the template arguments have, the messages can be confusing without quotes.

Reply:
Done